website/sidebars.json

```diff
@@ -9,25 +9,25 @@
     "MobX core": [
       "refguide/observable",
       "refguide/action",
-      "refguide/autorun",
-      "refguide/computed"
+      "refguide/computed",
```

Comment:
Note that this will require a small forwarding note in the explanation of `computed`, as it refers to `autorun` / things being observed.

Reply:
I've added it just before the example, pointing at autorun and the Reactions section. Does it resemble something you've had in mind, or was it a miss?